conda/plugins/virtual_packages/cuda.py

```diff
@@ -35,13 +96,15 @@ def cuda_version():
             "libcuda.so",  # check library path first
```

Comment:
I would also add `libcuda.so.1` here as that's what a typical `ldconfig` reports and is the `SONAME` in the actual `libcuda` binaries

Reply:
I add extra library search paths with `.1` suffix on Linux and macOS. On Windows, extra `nvcuda64.dll` is added.